content/fr/docs/setup/pick-right-solution.md

```diff
@@ -95,6 +95,8 @@ workloads, et plus encore.
 
 * [Oracle Container Engine for Kubernetes](https://docs.us-phoenix-1.oraclecloud.com/Content/ContEng/Concepts/contengoverview.htm) est un service entièrement géré, évolutif et hautement disponible que vous pouvez utiliser pour déployer vos applications conteneurisées dans le cloud.
 
+* [OVHcloud Managed Kubernetes Service](https://www.ovhcloud.com/fr/public-cloud/kubernetes/) est un service gratuit, entièrement géré, évolutif et hautement disponible pour héberger vos applications conteneurisées sur le Cloud Public OVHcloud. Il est disponible sur plusieurs continents.
```

Comment:
it's not completely free

Reply:
Like Google GKE or Azure AKS, OVHcloud offers the Managed Kubernetes Service tottaly for free (no cost for the kubernetes service, all the components are hosted for free, you dont pay any traffic or api calls). Of course, you pay the ressources that are orchestrated though kubernetes ( worker-only nodes, peristent volumes, external load balancer) at standard price. 

I think it is worth noting noting it is free, compared to paid alternatives in the same market such as AWS EKS.

Reply:
you can have that Google Kubernetes Engine it not indicated Free , and all others solutions there is no one indicated Free

Reply:
Indeed, thats a good point. I removed this information from my PR